docs/_guide/styles.md

```diff
@@ -110,9 +151,9 @@ class MyElement extends LitElement {
 }
 ```
 
-We strongly reccomend static styles, CSS custom properties, or [lit-html's `classMap` or `styleMap` directives](TODO) if you're stying non-host shadow root contents.
+TODO: Does changing `this.mainColro` reevaluate and restyle dynamically whenever `render()` is called? (my tests suggest no)
 
-## External stylesheets
+#### Define styles in an external stylesheet
```

Comment:
```suggestion
#### Load styles in an external stylesheet
```

"load" because the dev might be using someone else's styles. The critical bit is how they're loaded.